docs/plugins/irods_rule_language.md

```diff
@@ -550,6 +550,8 @@ The syntax of a rule with a nontrivial rule condition is as follows:
 }
 ~~~
 
+&#x26A0;&#xFE0F; **Use of `on()` may lead to unexpected behavior!** If all rule conditions for a rule fail, the rule will return an error! Use `if()` for simple conditions.
```

Comment:
Consider the `!!! Note` block notation for this pullout.

example...
https://github.com/irods/irods_docs/blob/95425578aa9bcdce629096f6aacd42387e957604/docs/release_notes.md?plain=1#L2061-L2062

```suggestion
 !!! Note
     Use of `on()` may lead to unexpected behavior. If all `on()` conditions within a rule fail, the rule will return an error. Consider use of `if()` for more familiar flow control.
```

Also consider whether this would be better placed at the beginning of the ```### Rule Condition``` section a little farther down - near line 590?

Reply:
Agreed. Even though `on()` is used here (and elsewhere, I'd imagine), it seems most appropriate to steer toward using `if()` in the conditional section.